lib/themes/dosomething/paraneue_dosomething/js/utilities/setting.js

```diff
@@ -0,0 +1,7 @@
+import dotty from 'dotty';
```

Comment:
Yup! It's a [nice little helper](https://www.npmjs.com/package/dotty) for getting nested properties from objects.